core/templates/pages/topic-viewer-page/practice-tab/practice-tab.component.html

```diff
@@ -24,7 +24,9 @@ <h2 class="tab-title" *ngIf="displayArea === 'topicViewer'">
       </mat-card-content>
       <mat-card-actions class="card-actions">
         <div class="fx-row fx-main-center fx-cross-center full-width">
-          <button mat-raised-button [disabled]="isStartButtonDisabled()" (click)="openNewPracticeSession()" class="practice-session-button protractor-test-practice-start-button">
+          <button mat-raised-button [disabled]="isStartButtonDisabled()"
+                  [ngClass]="{'practice-buttton-background-status-pending': !questionsStatusCallIsComplete, 'practice-buttton-background-status-complete': questionsStatusCallIsComplete}"
+                  (click)="openNewPracticeSession()" class="practice-session-button protractor-test-practice-start-button" >
```

Comment:
```suggestion
                  (click)="openNewPracticeSession()"
                  class="practice-session-button protractor-test-practice-start-button" >
```

Ditto at other places

Reply:
Applied to whole file!!
Done!